src/compaction/mod.rs

```diff
@@ -38,18 +37,20 @@ where
 {
     pub(crate) option: Arc<DbOption<R>>,
     pub(crate) schema: Arc<RwLock<Schema<R>>>,
-    pub(crate) version_set: VersionSet<R>,
+    pub(crate) version_set: Arc<VersionSet<R>>,
     pub(crate) manager: Arc<StoreManager>,
 }
 
+unsafe impl<R: Record> Send for Compactor<R> {}
```

Comment:
Should it only works on WASM target and we need to set a compilation condition on it?